.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "MD013": false,
+    "MD041": false
```

Comment:
What are these two about?

Reply:
13 is a warning about line length and 41 is about having content before the top-level heading: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md

If you don't have 41 then it complains about there being badges before the heading in README.md.